styles.css

```diff
@@ -62,7 +62,7 @@ body {
   /* Box Model */
   border-radius: 10px;
   margin-bottom: 10px;
-  padding: 30px 90px 30px 20px;
+  padding: 30px 90px 30px 20px !important;
```

Comment:
You want to avoid using !important, instead use a more specific selector or reorganize the order the rules appear in.